src/frontends/pytorch/src/frontend.cpp

```diff
@@ -60,17 +68,51 @@ FrontEnd::FrontEnd() : m_op_translators(get_supported_ops()) {}
 
 std::shared_ptr<Model> FrontEnd::convert(const InputModel::Ptr& model) const {
     auto converted_model = convert_partially(model);
-    normalize(converted_model);
-    std::set<std::string> unconverted_ops_types = get_unconverted_types_from_model(converted_model);
-    std::stringstream ops_str;
-    for (auto&& op_type : unconverted_ops_types) {
+    std::stringstream error_msg("Model wasn't fully converted.");
+    error_msg << "Model wasn't fully converted.";
+
+    std::string norm_err;
+    try {
+        normalize(converted_model);
+    } catch (const std::exception& e) {
+        norm_err = "\n-- normalize step failed with: " + std::string(e.what());
+    }
+
+    const auto& unconverted_ops = get_unconverted_types_from_model(converted_model);
+    std::stringstream unconverted_ops_msg;
+    std::stringstream failed_ops_msg;
+    std::stringstream failed_ops_short;
+    unconverted_ops_msg << "-- No conversion rule found for operations: ";
+    failed_ops_msg << " Failed operations detailed log:";
+    failed_ops_short << "-- Conversion is failed for: ";
+    bool at_least_one = false;
+    bool at_least_one_except = false;
+    for (auto&& op : unconverted_ops) {
         if (m_telemetry) {
-            m_telemetry->send_event("error_cause", "pytorch_" + op_type);
+            m_telemetry->send_event("error_cause", "pytorch_" + op.first);
+        }
+        if (op.second.empty()) {
+            if (at_least_one)
+                unconverted_ops_msg << ", ";
+            unconverted_ops_msg << op.first;
+            at_least_one = true;
+        } else {
+            if (at_least_one_except)
+                failed_ops_short << ", ";
+            failed_ops_short << op.first;
+            failed_ops_msg << "\n-- " << op.first << " with a message:\n" << op.second;
```

Comment:
Also recommend to make an indentation for paragraphs. As you are printing basically a list of issues, it is hard to find where one ends and the next starts. `--` is a good try, but as I see this can be better.

Also please use more blank lines as a separator (in addition to indentation or alone). Output may include multiple python stack traces (as I see in your output example) which itself contains black lines as a separators, and provide another level of separation of items.

Reply:
I personally think that less newlines is better, otherwise it just occupy more space with same information

Reply:
Indentation was not a good idea indeed without knowing desirted "screen width", so OK, let's leave it as-is.